modulefiles/module_base.wcoss_dell_p3

```diff
@@ -23,10 +23,14 @@ module load pm5
 
 #module load HDF5-serial/1.10.1
 #module load NetCDF/4.5.0
-module use /usrx/local/nceplibs/dev/NCEPLIBS/modulefiles
+#module use /usrx/local/nceplibs/dev/NCEPLIBS/modulefiles
+#module load hdf5_parallel/1.10.6
+#module load netcdf_parallel/4.7.4
+#module load esmf/8.0.0_ParallelNetCDF
+module use /gpfs/dell2/emc/modeling/noscrub/emc.nemspara/soft/modulefiles
```

Comment:
Does NCEPLIBS have the modules below?   If not, they need to install these modules in their space and then work to get these modules installed in operations.

Reply:
@junwang-noaa please help me identify what can be removed and what needs to be changed in this file.

Reply:
I removed commented out lines in module file. Will commit.